actioncable/lib/action_cable/subscription_adapter/test.rb

```diff
@@ -0,0 +1,24 @@
+module ActionCable
+  module SubscriptionAdapter
+    # == Test adapter for Action Cable
+    class Test < Base
```

Comment:
Yes, of course. I'll add some docs to this class and to the helper after we decide that everything is ok.